server.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import app from './app';
+
+dotenv.config();
+
+// connect local db for local development
+const DB = process.env.DATABASE_LOCAL_URL;
+
+mongoose
```

Comment:
Would be more readable if you can separate this DB logic on another file, eg a dbConfig then just call the function in a start server function

Reply:
ok, sure